build/gitkubed/Dockerfile

```diff
@@ -4,10 +4,10 @@ MAINTAINER tiru@hasura.io
 # Install openssh server
 RUN apt-get update \
  && apt-get install -y upx-ucl binutils curl openssh-server git jq \
- && curl -o /tmp/docker-1.12 'https://get.docker.com/builds/Linux/x86_64/docker-1.12.6.tgz' \
- && tar -xf /tmp/docker-1.12 -C /tmp \
+ && curl -o /tmp/docker-17.05 'https://get.docker.com/builds/Linux/x86_64/docker-17.05.0-ce.tgz' \
```

Comment:
I would discourage using docker 17.05 (that was an edge release, and does have some known vulnerabilities - given, those were in the daemon, not the CLI); current versions of docker can be found on download.docker.com (get.docker.com is no longer being updated with new versions); static amd64 (x86_64) binaries can be found (e.g.) here; https://download.docker.com/linux/static/stable/x86_64/